### ISSUE TITLE
[URL standards] URL doesn't accurately reflect the location of the page within the site.  (07.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54292]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
URL doesn't accurately reflect the location of the page within the site. 

### Issue Details
Several links on VA side go to TRICARE page version instead of VA version. The ones I identified are as follows: 1.) Contact Us page > fix both the Medical records office link & Billing and insurance link. 2.) Lovell homepage > fix both the “access your health records” link and the "check your billing, insurance, and payment options" link (both in blue background box) 

### VA.gov Experience Standard
[Category Number 07, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Swap these TRICARE links for the VA version.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by davidmpickett** [comment]
This is still a problem:
https://www.va.gov/lovell-federal-health-care-va/contact-us/

**Comment by davidmpickett** [comment]
Seems like it might be a consequence of https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11215

**Comment by stefaniefgray** [comment]
**Updates:**
Broken link scanner found that the Spotlight section for Lovell Tricare locations was linking to `/lovell-federal-tricare-health-care/about-us/performance`

This created broken links across the following pages:
- https://prod.cms.va.gov/lovell-federal-health-care-tricare
- https://prod.cms.va.gov/lovell-federal-health-care-tricare/locations/captain-james-a-lovell-federal-health-care-center
- https://prod.cms.va.gov/lovell-federal-health-care-tricare/locations/uss-osborne-dental-clinic
- https://prod.cms.va.gov/lovell-federal-health-care-tricare/locations/uss-red-rover
- https://prod.cms.va.gov/lovell-federal-health-care-tricare/locations/uss-tranquillity
- https://prod.cms.va.gov/lovell-federal-health-care-tricare/locations/zachary-and-elizabeth-fisher-medical-and-dental-clinic

I've since updated the link (listed within the "Additional Details About This Region" section of the main `/lovell-federal-health-care-tricare` page), which in turn updates it across all of the pages listed above.

**New dilemma:**

The main `/lovell-federal-health-care-tricare` page is reporting that it still has 6 unpublished pages linked from it:

1. [Homeless Veteran care at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/homeless-veteran-care-at-lovell-federal-health-care-tricare
2. [Minority Veteran care at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/minority-veteran-care-at-lovell-federal-health-care-tricare
3. [Palliative and hospice care at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/palliative-and-hospice-care-at-lovell-federal-health-care-tricare
4. [Caregiver support at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/caregiver-support-at-lovell-federal-health-care-tricare
5. [Geriatrics at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/geriatrics-at-lovell-federal-health-care-tricare
6. [MOVE! weight management at Lovell Federal health care - TRICARE] https://prod.cms.va.gov/lovell-federal-health-care-tricare/health-services/move-weight-management-at-lovell-federal-health-care-tricare

These services are currently missing from the following page: https://www.va.gov/lovell-federal-health-care-tricare/health-services/

cc: @swirtSJW @omahane @davidmpickett It looks like the Lovell editors archived these nodes on purpose, and since they're not listed on the Tricare "Health Services" page, I'm confused about where the "link to unpublished page" alerts are coming from.

Please advise next week when we're back in office -- thank you!

**Comment by jilladams** [comment]
From scrum: 

re: New dilemma: 
* The entity references are dynamically injected here, not editable in the node. In the front-end, @ryguyk will verify that the front-end won't display unpublished content. 
* Nicholas and Katy from Lovell archived these pages. Helpdesk to follow up re: whether they should be deleted.

Could use a CMS issue: Archived content shouldn't show up on CMS nodes in the entity reference list for broken links. Swirt to make this ticket.

**Comment by ryguyk** [comment]
Confirming that the front end will only display published services.
### Code
The query specifically limits to `status === 1`:
```
reverseFieldRegionPageNode(limit: 500, filter: {conditions: [{field: "type", value: "regional_health_care_service_des"}, {field: "status", value: "1", operator: EQUAL}]}) {...
```

### Result
The section "Social programs and services" does not include "Homeless veteran care".
![image](https://user-images.githubusercontent.com/6863534/230953182-d1522362-d978-4758-b283-6ca8327c15f6.png)

**Comment by stefaniefgray** [comment]
Waiting on reply from Nicholas/Katie @ Lovell

**Comment by stefaniefgray** [comment]
@ryguyk "Yes, we think it would be alright to delete those items since they aren’t currently in use for our TRICARE patients. Thanks!"

**Comment by jilladams** [comment]
@stefaniefgray is next step for you/Troy,or Katie/Nicholas, or someone from Fac to archive those?

**Comment by stefaniefgray** [comment]
@jilladams Member of Facilities team can delete 👍

**Comment by jilladams** [comment]
@stefaniefgray scrum discussion today: we missed a beat somewhere along the way in suggesting that these pages can / should be deleted. In general: we don't delete content. Archiving should be sufficient, and because it's not here, the right way to fix is to get  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13214 prioritized to actually _solve_ the problem.  So: we are not going to delete this content, based on today's discussion. Apologies for sending you off to chase a rabbit trail. 

Speaking of rabbit trails: this whole menu / broken links thing is orthogonal to the nature of the ticket here. For the actual issue Shira reported:  @kmariepat-cityfriends we need to work out whether this one goes to VHADM or we need a tracking ticket for work against it in the Facilities board.

**Comment by kmariepat-cityfriends** [comment]
@jilladams I am prioritizing Archived services should not result in node link report notices on system pages #13214 in the upcoming sprint (sprint 83) 

I am unclear on what additional action is needed from facilities of VHA DM, I will bring up in sprint planning
